app/controllers/application_controller.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  include ClientSideApiHelper
```

Comment:
I had to manually add this - do you think this is something that onboardees should try to find on their own or should we just include it from the start?

Reply:
I think it should definitely be included from the start.  Good catch 🎣 

Another improvement you could make to `Launchpad` if you're feeling up to it